### DISCUSSION TITLE
Unused genres

### DISCUSSION THREAD

**Discussion by kidwellj** [advplyr/audiobookshelf#1034] [General]
I've been cleaning up metadata on podcasts in my library, but even when genres are no longer populated by any files, they persist in the listing. Is there some way I'm missing to clear out unused genres?

**Comment by advplyr** [comment]
Even after you refresh the browser they are still there?

**Reply by kidwellj** [reply]
It was definitely happenning after a refresh, but I've been unable to replicate the issue since then! Will report back if I can manage to do so.

**Reply by advplyr** [reply]
Sounds good, the genres are not a separate entity in abs. They are just pulled from each library item each request.
